### PR TITLE
add image to meta tag

### PR DESCRIPTION
**Pull request by ethulia**
Pretty new to this, but an attempt at fixing https://github.com/Growstuff/growstuff/issues/733

Let me know if the _meta.html file isn't the right place to put the tag!

### CODE REVIEW

**Comment by pozorvlak** [comment]
Looks good, and I think `_meta.html` is exactly the right place! Could you add a test for the image's presence? We're phasing out view tests in general, but I think it would be fine in this case to add another test to `views/layouts/_meta_spec.rb` along the lines of the existing ones.

[If the answer's "no, I'm not that comfortable with Ruby yet", no problem, we'd be happy to help you out!]

**Comment by ethulia** [comment]
Sure, will add the test!

**Comment by pozorvlak** [comment]
Great!

**Comment by ethulia** [comment]
I can't seem to get the tests to run locally, so not sure if I can write the test. Can someone else add it in?

**Comment by pozorvlak** [comment]
Sure, but I'd like to work out why you can't get the tests to run. Can you copy-and-paste the command you ran and the output you got here?

**Comment by Skud** [comment]
Hi @ethulia, this looks good to me :)  But as @pozovlak says, can you let us know which tests are failing for you so we can help you get that sorted out?  Also, please add yourself to CONTRIBUTORS.md!

File: app/views/layouts/_meta.html.haml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 %head
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta property="og:image" content="meta_image.png" />
```
Comment (Skud) [inline app/views/layouts/_meta.html.haml]: this is a small thing, but perhaps we could rename this something like growstuff_logo_ogimage.png or something?  "meta_image" doesn't really tell us what it's for.

**Comment by pozorvlak** [comment]
It looks like we merged someone else's fix to this issue, in #746 - sorry! Closing this as no longer needed :-(

**Pull request closed without merging**